<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinSaver</class>
 <widget class="QDialog" name="pinSaver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change PIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">text-color = red;</string>
  </property>
  <widget class="QLineEdit" name="pinEdit2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cofirm New PIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pinEdit1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter New PIN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorMsg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>* Fields do not match</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
